### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
add more entries for "deploy" action.
group web app runtime by platform and java version.

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorSingleTab.java [hunk 1]
```diff
@@ -64,11 +64,15 @@ private void initListener() {
...
-        this.monitorLogTablePanel.addRunActionListener(e -> Optional.ofNullable(this.parentView.getSelectedWorkspace()).ifPresentOrElse(t -> loadLogs(),
+        this.monitorLogTablePanel.addRunActionListener(e -> Optional.ofNullable(this.parentView.getSelectedWorkspace()).ifPresentOrElse(t -> loadLogsTriggeredByUser(),
                 () -> AzureMessager.getMessager().info(message("azure.monitor.info.selectWorkspace"), null, selectWorkspaceAction())));
     }
 
     @AzureOperation(name = "user/monitor.execute_query")
+    private void loadLogsTriggeredByUser() {
+        loadLogs();
+    }
+
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorSingleTab.java]: add byToolkit telemetry

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorTabbedPane.java [hunk 2]
```diff
@@ -70,6 +70,12 @@ public void selectTab(String tabName) {
         this.initResourceId = null;
     }
 
+    @AzureOperation(name = "user/monitor.change_table_tab", params = {"tabName"})
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorTabbedPane.java:73]: check if tabName in our list

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorSingleTab.java [hunk 3]
```diff
@@ -64,11 +64,15 @@ private void initListener() {
             }
             this.monitorLogDetailsPanel.setViewText(viewerTitle, viewerText);
         });
-        this.monitorLogTablePanel.addRunActionListener(e -> Optional.ofNullable(this.parentView.getSelectedWorkspace()).ifPresentOrElse(t -> loadLogs(),
+        this.monitorLogTablePanel.addRunActionListener(e -> Optional.ofNullable(this.parentView.getSelectedWorkspace()).ifPresentOrElse(t -> loadLogsTriggeredByUser(),
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorSingleTab.java]: add telemetry at action listener

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/component/Node.java [hunk 4]
```diff
@@ -187,6 +182,14 @@ public void dispose() {
         this.view.dispose();
     }
 
+    private void updateInlineActions() {
```
Comment (ivywei0125) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/component/Node.java]: revert commit

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/appservice/platform/RuntimeComboBox.java [hunk 5]
```diff
@@ -61,20 +56,13 @@ private void setGroupRender() {
...
     private String getSeparatorCaption(Runtime item) {
-        String key = StringUtils.EMPTY;
-        for (final Map.Entry<String, List<Runtime>> entry : runtimeSeparatorMap.entrySet()) {
-            key = entry.getKey();
-            if (entry.getValue().contains(item)) {
-                break;
-            }
-        }
-        return key;
+        return String.format("%s & %s", item.getOperatingSystem().toString(), item.getJavaVersion().toString());
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/appservice/platform/RuntimeComboBox.java:65]: check lower or upper

**Review by wangmingliang-ms — APPROVED** [review]
LGTM, remember to check group name

**Pull request merged**